model/const.go

```diff
@@ -162,6 +162,8 @@ const (
 
 	// GetApprovalPath defines get-approval path style suffix
 	GetApprovalPath = "/greenfield/admin/v1/get-approval"
+	// UniversalEndpointPath defines universal endpoint path style suffix
+	UniversalEndpointPath = "/download/{bucket:[^/]*}/{object:.+}"
```

Comment:
Put the regular expressions into the router?

Reply:
For the static part /download/, is it ok to also put this part in router too?